qiskit/__init__.py

```diff
@@ -31,6 +31,7 @@
 from qiskit.circuit import QuantumCircuit
 from qiskit.execute import execute
 from qiskit.compiler import transpile, assemble
+from qiskit.scheduler import schedule
```

Comment:
@taalexander When I include this locally, I can no longer import `Aer` or `IBMQ`... when I remove it, I have no problem. Do you know what might be the problem? I can't see what the issue would be

Reply:
This is due to cyclic import.
https://github.com/Qiskit/qiskit-terra/blob/11fe607fa8b5fbb118a26d7fb6ba54d8a4fb9f04/qiskit/scheduler/schedule.py#L23
You can remove hinting or change import order so that it comes after providers.